contracts/TiramisuSavingsClub.sol

```diff
@@ -116,6 +117,7 @@ contract TiramisuSavingsClub {
         uint _groupId = getGroupId();
         Group storage _group = groups[_groupId];
         require(msg.sender == _group.members[_group.ownerIndex], "Caller is not the group owner");
+        string memory _ownerName = memberNames[msg.sender];
```

Comment:
Is this line necessary? This copies this value from storage into memory.

L134 could replace ```_ownerName``` with ```memberNames[msg.sender]``` to save on gas

Reply:
I put it there to accommodate the event sent at the very end of the function, because the memberNames object gets deleted beforehand.